datumaro/components/dataset.py

```diff
@@ -114,6 +114,9 @@ def get_subset(self, name):
     def subsets(self):
         return self.data
 
+    def get_annotated_size(self):
+        return sum(bool(s.annotations) for s in self._traversal_order.values())
+
```

Comment:
Is it duplicated with the below one?

Reply:
It is not duplicated one, `get_annotated_size` function which is not in `DatasetItemStorage` is to use it.